sqlmesh/core/loader.py

```diff
@@ -280,6 +312,7 @@ def _load() -> Model:
                         physical_schema_override=config.physical_schema_override,
                         project=config.project,
                     )
+                    return with_mocked_dependencies(loaded_model)
```

Comment:
So we're going to cache a model rendered with mocked dependencies? Doesn't this defeat the purpose of the caching?

Reply:
as long as we always calculate the hash with mock it should be ok